tenants-certified-attributes-updater/src/main/scala/it/pagopa/interop/tenantscertifiedattributesupdater/util/Utils.scala

```diff
@@ -61,6 +61,12 @@ object Utils {
     tenants: List[PersistentTenant],
     attributesIndex: Map[UUID, AttributeInfo]
   ): TenantActions = {
+
+    val admittedAttributeOrigins: List[String] = institutions.map(_.origin).distinct
```

Comment:
LGTM, it just took me some time to figure out that these institutions are only from the party registry, and then why it makes sense to admit these origins.
A comment or a variable rename might help